src/libratbag-private.h

```diff
@@ -215,6 +224,16 @@ struct ratbag_resolution{
 	uint32_t capabilities;
 };
 
+struct ratbag_led {
+	struct ratbag_profile *profile;
+	enum ratbag_led_type type;
+	enum ratbag_led_mode mode;  /**< off; on; cycle; breath  */
+	struct ratbag_color color; /**< led color  */
+	unsigned int rate;	/**< rate of action 100 - 20000 */
+	unsigned int bright;	/**< brightness of the LED 1 - 100 */
```

Comment:
the brightness at the lowest level is 0 - 100, so its a percentage. so 0..255 won't fit here